app/models/grocery_store.rb

```diff
@@ -0,0 +1,7 @@
+class GroceryStore < ActiveRecord::Base
+  validates_presence_of :lat, :lng, :name, :place_id
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.